modules/local/fusioncatcher/detect/main.nf

```diff
@@ -2,8 +2,7 @@ process FUSIONCATCHER {
     tag "$meta.id"
     label 'process_high'
 
-    conda "${moduleDir}/environment.yml"
```

Comment:
Modules that do not use conda typically have a warning if the user launches with `-profile conda`.

Reply:
See example at: 

https://github.com/nf-core/demultiplex/blob/master/modules/nf-core/bclconvert/main.nf#L25

Reply:
Well, we do want to warn the user that conda is not usable everywhere, so that is desired behaviour